cpp/oneapi/dal/table/test/csr.cpp

```diff
@@ -352,6 +417,59 @@ TEST("can construct table from arrays and share the ownership of the data with t
     REQUIRE(t.get_non_zero_count() == element_count);
 }
 
+#ifdef ONEDAL_DATA_PARALLEL
+
+TEST((std::string("can construct table from arrays holding the data allocated on the device") +
+      std::string(" and share the ownership of the data with those arrays"))
+         .c_str()) {
```

Comment:
```suggestion
TEST("can construct table from arrays holding the data allocated on the device"
         " and share the ownership of the data with those arrays") {
```

Reply:
This does not compile. And I do not aware about the better way of writing multiline captions in the TEST macros.